### DISCUSSION TITLE
Is it really secure to run Flow Launcher?

### DISCUSSION THREAD

**Discussion by recepbalibey** [Flow-Launcher/Flow.Launcher#1815] [Q&A] [answered]
Virustotal:
![image](https://user-images.githubusercontent.com/47719328/213259169-65c59bba-8a36-4d81-9831-c4e6bf9eb5e3.png)

I just installed the .exe from here:
![image](https://user-images.githubusercontent.com/47719328/213259291-ab474d1f-628c-4741-aefd-557be7e33e50.png)
And when I click to run:
![image](https://user-images.githubusercontent.com/47719328/213259347-b4f35ee6-4952-4775-ba89-5d5e4d8db4ab.png)

I don't get it.

**Comment by Garulf** [comment, marked as answer]
> Virustotal:
> 
> ![image](https://user-images.githubusercontent.com/47719328/213259169-65c59bba-8a36-4d81-9831-c4e6bf9eb5e3.png)
> 
> 
> 
> I just installed the .exe from here:
> 
> ![image](https://user-images.githubusercontent.com/47719328/213259291-ab474d1f-628c-4741-aefd-557be7e33e50.png)
> 
> And when I click to run:
> 
> ![image](https://user-images.githubusercontent.com/47719328/213259347-b4f35ee6-4952-4775-ba89-5d5e4d8db4ab.png)
> 
> 
> 
> I don't get it. 

The windows dialog just means Flow  does not have a signed certificate. Which as long as you download directly from our site won't be an issue. 

The virus total false positives are a known issue and we have taken steps to contact virus total about the issue.
